File: web-shop-backend/src/main/java/com/shadow_garden/webshopbackend/dto/product/SellProductRequest.java
package com.shadow_garden.webshopbackend.dto.product;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SellProductRequest {
    private long productId;
    private long sellerId;
    private int quantity;
}

File: web-shop-frontend/model/entity/product/SellProductRequest.cs
using Newtonsoft.Json;

namespace web_shop_frontend.model.entity.product;

public class SellProductRequest
{
    [JsonProperty("productId")] public long ProductId { get; set; }
    [JsonProperty("sellerId")] public long SellerId { get; set; }
    [JsonProperty("quantity")] public int Quantity { get; set; }

    public SellProductRequest(long productId, long sellerId, int quantity)
    {
        ProductId = productId;
        SellerId = sellerId;
        Quantity = quantity;
    }
}